### PR TITLE
Transcribe Hypercube to javascript.

### PR DESCRIPTION
**Pull request by FriskyDev**
Would love to create a better matrix math library or use one of the existing ones. The .array() function, and set() operators of the p5.Vector would seem to make this easier, but the arrays are one dimensional, so it really feels like there needs to be a matrix class, with corresponding conversions. (and I am thinking out loud: inside p5.js)

I say this because I used similar, **but slightly different code** for another one of the samples. Then I used that one to create **a slightly different one** I hooked up to the Barnsley Fern example (which was doing the algebraic equivalent).

### CODE REVIEW

**Comment by FriskyDev** [comment]
There error is from some new check?

**Comment by FriskyDev** [comment]
🥇 thanks @gruselhaus

File: CodingChallenges/CC_113_Hypercube/P5/matrix.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+// Javascript transcription: Chuck England
```
Comment (shiffman) [inline CodingChallenges/CC_113_Hypercube/P5/matrix.js]:
```suggestion
// JavaScript transcription: Chuck England
```

File: CodingChallenges/CC_113_Hypercube/P5/sketch.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+// Daniel Shiffman
+// http://youtube.com/thecodingtrain
+// http://codingtra.in
+// Javascript transcription: Chuck England
```
Comment (shiffman) [inline CodingChallenges/CC_113_Hypercube/P5/sketch.js]:
```suggestion
// JavaScript transcription: Chuck England
```

File: CodingChallenges/CC_113_Hypercube/P5/P4Vector.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// Daniel Shiffman
+// http://youtube.com/thecodingtrain
+// http://codingtra.in
+// Javascript transcription: Chuck England
```
Comment (gruselhaus) [inline CodingChallenges/CC_113_Hypercube/P5/P4Vector.js]:
```suggestion
// JavaScript transcription: Chuck England
```
Here is the last one. Then we can merge :D

**Comment by gruselhaus** [comment]
@FriskyDev Thank you ❤️

**Pull request merged**